test/cypress/integration/web-payments/apple-pay.spec.js

```diff
@@ -196,5 +204,42 @@ describe('Apple Pay payment flow', () => {
       cy.get('#enter-card-details-container').should('be.visible')
       cy.get('#card-no').should('be.visible')
     })
+
+    it('Should setup the payment and load the page', () => {
```

Comment:
why is this test necessary?

Reply:
Because between each test we need to reset the view

Reply:
I guess this makes it clear, it would be weird not to describe what a part of the test is doing